src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -0,0 +1,198 @@
+        [InlineData(1000000, 64, 64)]
+        [InlineData(1000000, 256, 256)]
+        [InlineData(1000000, 1024, 1024)]
+        [InlineData(100000, 4096, 4096)]
+        [InlineData(100000, 16384, 16384)]
+        [InlineData(100000, 65536, 65536)]
+        public void Add(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Add(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 64)]
+        [InlineData(1000000, 256, 256)]
+        [InlineData(1000000, 1024, 1024)]
+        [InlineData(100000, 4096, 4096)]
+        [InlineData(100000, 16384, 16384)]
+        [InlineData(100000, 65536, 65536)]
+        public void Subtract(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Subtract(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 64)]
+        [InlineData(1000000, 256, 256)]
+        [InlineData(100000, 1024, 1024)]
+        [InlineData(10000, 4096, 4096)]
+        [InlineData(1000, 16384, 16384)]
+        [InlineData(100, 65536, 65536)]
+        public void Multiply(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Multiply(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16)]
+        [InlineData(1000000, 64)]
+        [InlineData(1000000, 256)]
+        [InlineData(100000, 1024)]
+        [InlineData(10000, 4096)]
+        [InlineData(1000, 16384)]
+        [InlineData(100, 65536)]
+        public void Square(int count, int bits)
+        {
+            RunBenchmark(count, bits, v => BigInteger.Multiply(v, v));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 16)]
+        [InlineData(1000000, 256, 128)]
+        [InlineData(100000, 1024, 512)]
+        [InlineData(10000, 4096, 2048)]
+        [InlineData(1000, 16384, 8192)]
+        [InlineData(100, 65536, 32768)]
+        public void Divide(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Divide(l, r));
+        }
+
+        [Theory]
+        [ActiveIssue("PerformanceTest")]
+        [InlineData(1000000, 16, 16)]
+        [InlineData(1000000, 64, 16)]
+        [InlineData(1000000, 256, 128)]
+        [InlineData(100000, 1024, 512)]
+        [InlineData(10000, 4096, 2048)]
+        [InlineData(1000, 16384, 8192)]
+        [InlineData(100, 65536, 32768)]
+        public void Remainder(int count, int leftBits, int rightBits)
+        {
+            RunBenchmark(count, leftBits, rightBits, (l, r) => BigInteger.Remainder(l, r));
```

Comment:
For Divide and Remainder, is there any concern that we could be getting unlucky with the chosen values so that the numerator is frequently smaller than the denominator and we don't really exercise the main code paths?